modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -90,8 +103,40 @@ const HeadlessResourceItem = ({
 	);
 };
 
+interface IProviderRendererProps {
+	headlessResourcesMap: Map<String, THeadlessResource>;
+	itemData: TFDSEntry;
+}
+
+const ProviderRenderer: React.FC<IProviderRendererProps> = ({
+	headlessResourcesMap,
+	itemData,
+}: IProviderRendererProps) => {
+	const headlessResource = headlessResourcesMap.get(itemData.entityClassName);
+
+	return (
+		<>
+			{`${headlessResource?.name} (${headlessResource?.bundleLabel} ${headlessResource?.version})`}
+		</>
```

Comment:
nit: are the fragment wrappers(`<>`) necessary here? I think I remember them being required in some strange cases, but don't fully remember